content/en/setup/_index.md

```diff
@@ -9,7 +9,7 @@ menu:
 
 ## Command-line interface tool
```

Comment:
```suggestion
## Command line interface tool
```
It also wouldn't hurt to if we simplified the title and used, e.g., "CLI tool". Or we leave out the "interface" as in the phrase below.